### DISCUSSION TITLE
imodeljs-frontend error

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#954] [Q&A]
load some imodel file success 
but some imodel file error.and web brower console this:
Error   |imodeljs-frontend.RpcInterface| Not Found: TreeId=17_0-E:0_0x13 TileId=-3-1-0-0-0-1 {"message":"TreeId=17_0-E:0_0x13 TileId=-3-1-0-0-0-1","errorNumber":65574,"name":"Not Found","isError":true,"ExceptionType":"BackendError","ActivityId":"43a92cc6-6648-4c4a-a752-5b42044d2070","SessionId":"00000000-0000-0000-0000-000000000000","ApplicationId":"","ApplicationVersion":""}

**Comment by pmconne** [comment]
Does this occur when opening a 2d view, or a 3d view? If 3d, it's not a big deal - it means you have an invalid model Id in your model selector.

**Reply by paprophet** [reply]
3d view,
some time console this error

**Reply by pmconne** [reply]
Is the view displaying as expected? Is the console message the only problem? If so, you can ignore it, or you can look in your iModel to see if you have a 3d model with Id 0x17. If you don't have one, fix your model selector to not include that model Id.

**Reply by paprophet** [reply]
usually,the error frequent occurrence,when this error show,the web  load none about the bim file, but open other bim file is success load ,and if i restart the imodel backend,this bim file load success and not show this error !
my code use  https://github.com/imodeljs/imodeljs-samples/tree/master/interactive-app/ninezone-sample-app
and i will send the video to your email! thank you!

**Reply by paprophet** [reply]
window.addEventListener("beforeunload", imodelclose);
if i close the page tab or close the google chrome ,the imodelclose method not run .
i want the google web brower close the imodel web page or close the google web brower, the imodel will close and the backend will close process about this imodel link !

**Reply by pmconne** [reply]
If your browser's beforeunload event is not behaving as you expect, please contact the developer of your browser.
If `imodelclose` does something `async` (such as close an IModelConnection), I wouldn't be surprised if the browser does not wait for that async function to complete.

It is not generally the responsibility of a web app's frontend to close an iModel on the backend. Web apps are designed to support multiple users accessing the same backend.
If your app involves a backend and frontend running on the same device consider writing an electron app instead.

**Reply by paprophet** [reply]
when the googel web brower console this error,the imodel bim file is lock by imodeljs backend,and any other pc google brower open this model and not show the model, and console this error too.
if i restart the backend,the error  not show and imodeljs can open ,but some time the error show . 
i want know why show this error,and how can solve this problem ,

![QQ图片20210521154122](https://user-images.githubusercontent.com/18022720/119101374-7e58e280-ba4b-11eb-8eae-e53b9230fdd8.png)


![QQ截图20210521160355](https://user-images.githubusercontent.com/18022720/119103861-38514e00-ba4e-11eb-8fff-0d420096fbaa.jpg)
